Formula/ghc.rb

```diff
@@ -48,6 +49,24 @@ class Ghc < Formula
   end
 
   def install
+    # As of Xcode 7.3 (and the corresponding CLT) `nm` is a symlink to `llvm-nm`
+    # and the old `nm` is renamed `nm-classic`. Building with the new `nm`, a
+    # segfault occurs with the following error:
+    #   make[1]: * [compiler/stage2/dll-split.stamp] Segmentation fault: 11
+    # Reported 19th Apr 2016 here: https://ghc.haskell.org/trac/ghc/ticket/11951
+    if ENV.compiler == :clang && MacOS.clang_build_version >= 703
+      mkdir "brewtools"
```

Comment:
FWIW you don't need to do this explicitly; `.write` below will create the path if it doesn't exist.